lib/transform.js

```diff
@@ -0,0 +1,81 @@
+const vegadataflow = require('../vendor/vega-dataflow.js');
```

Comment:
Is there reason that we are using mixed syntax of ES6 and ES5. 
Eg: `const` is ES6 and `require` - ES5. Why not `import` ?